migrations/migrations/20210406205811-name-of-migration.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+      return queryInterface.createTable('user', {
+        id: {
+          type: Sequelize.INTEGER,
+          unique: true,
+          autoIncrement: true,
+          primaryKey: true,
+          allowNull: false
+        },
+        firstName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        lastName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        email: {
+          type: Sequelize.STRING,
+          unique: true,
+          allowNull: false
+        },
+        password: {
+          type: Sequelize.STRING,
+          allowNull: false
+        }
+      }, {
+        timestamps: true
```

Comment:
I think this property ` timestamps: true` is not correct in the migration this goes in the model

Reply:
property removed